client/asset/eth/node.go

```diff
@@ -224,6 +224,7 @@ func startNode(node *node.Node, network dex.Network) (*les.LightEthereum, error)
 	}
 
 	ethCfg.SyncMode = downloader.LightSync
+	ethCfg.RPCTxFeeCap = 0
```

Comment:
I should add a comment.

Reply:
Added https://github.com/decred/dcrdex/compare/3f016ca304b5a7da9e6b7470247c8aee95a17157..16184b74213accddb30fd34f94e06823ce877744